sonar-slang-plugin/src/test/java/com/sonarsource/slang/kotlin/KotlinConverterTest.java

```diff
@@ -140,13 +125,15 @@ public void testFunctionDeclaration() {
     assertThat(constructorFunction.modifiers()).isEmpty();
     assertTree(constructorFunction.returnType()).isNull();
     assertThat(constructorFunction.formalParameters()).hasSize(2);
+    assertTree(constructorFunction).hasParameterNames("a", "b");
     assertTree(constructorFunction.body()).isNull();
 
     FunctionDeclarationTree emptyLambdaFunction = (FunctionDeclarationTree) kotlin("{ }");
     assertTree(emptyLambdaFunction.name()).isNull();
     assertThat(emptyLambdaFunction.modifiers()).isEmpty();
     assertTree(emptyLambdaFunction.returnType()).isNull();
     assertThat(emptyLambdaFunction.formalParameters()).isEmpty();
+    assertTree(functionWithPrivate).hasParameterNames();
```

Comment:
`functionWithPrivate` is not the correct variable here. Also, this assert will not do anything special